Makefile

```diff
@@ -96,12 +96,6 @@ format: $(GOIMPORTS)
 # ------------------------------------------------------------------------------
 #  dependencies
 
-.PHONY: bootstrap
-bootstrap: vendor
-
-$(DEP):
-	go get -u github.com/golang/dep/cmd/dep
-
 $(GOX):
 	go get -u github.com/mitchellh/gox
```

Comment:
In module-aware mode, this will add `gox` as a dependency to `go.mod` which is probably not desired. Could be fixed like this:

`GO111MODULE=off go get -u github.com/mitchellh/gox`

Reply:
On the contrary, I think adding `gox` as a dependency to `go.mod` is ideal, as it guarantees that every developer is using the same version of the tool. See [here](https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module) for a more detailed explanation on how this would work.

Reply:
Interesting. Thanks for the link. I wasn't aware of that. Then, I guess, a `tools.go` file should be added as suggested.